pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -34,8 +34,9 @@ type Build struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   BuildSpec   `json:"spec"`
-	Status BuildStatus `json:"status"`
+	Spec    BuildSpec   `json:"spec"`
+	Status  BuildStatus `json:"status"`
+	Timeout string      `json:"timeout,omitempty"`
```

Comment:
I think we'll want `Timeout` to exist inside of `BuildSpec`

Reply:
That makes sense. 👍